molgenis-security/src/main/java/org/molgenis/security/permission/MolgenisPermissionController.java

```diff
@@ -1,25 +1,52 @@
 package org.molgenis.security.permission;
 
+import com.google.common.collect.ImmutableMap;
+import org.molgenis.auth.Role;
+import org.molgenis.data.DataService;
+import org.molgenis.data.Entity;
+import org.molgenis.data.QueryRule;
+import org.molgenis.data.i18n.LanguageService;
+import org.molgenis.data.meta.model.EntityType;
+import org.molgenis.data.security.acl.*;
+import org.molgenis.data.support.QueryImpl;
 import org.molgenis.security.core.PermissionService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
 
 import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.molgenis.auth.RoleMetadata.ROLE;
+import static org.molgenis.data.QueryRule.Operator.SEARCH;
 import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.PUT;
 
 @Controller
 @RequestMapping("/permission")
 public class MolgenisPermissionController
 {
 	private final PermissionService molgenisPermissionService;
+	private final EntityAclService entityAclService;
+	private final EntityAclManager entityAclManager;
```

Comment:
@fdlk @tommydeboer EntityAclManager is a EntityAclService, so Autowiring EntityAclManager is sufficient